src/utils/logging.js

```diff
@@ -22,12 +24,53 @@ import config from '../config';
 //   * An `Error` argument formats with (the name + message and) just the
 //     single source location where it was created, no stack trace.
 
+/**
+ * Log an event (a string or Error) at some arbitrary severity.
+ *
+ * The error will be logged to Sentry, including a stack trace. The stack trace
+ * is taken from `err` if an `Error` object, and otherwise synthesized from the
+ * call site.
+ *
+ * To match the function name, the default severity is 'log'.
+ *
+ * Returns a Sentry event_id, although this is not expected to be useful.
+ */
+export const logToSentry = (event: string | Error, level: SeverityType = Severity.Log): string => {
```

Comment:
I'd like to keep this unexported, for basically the reason described in fd8f5cdae:

    logging: Migrate all `logErrorRemotely` calls to new `logging.*` API.
    
    Two reasons to get "remotely" out of the function's name:
    
     * It was already inaccurate -- this function was just as much for
       logging locally as remotely.
    
     * If accurate, it wouldn't belong in the API.  The choice that
       makes sense to express at individual error sites is between
       message severity levels (error, warning, etc.), while specific
       reporting routes (local logs, Sentry, etc.) are a policy choice
       to make centrally per severity level.

Naturally we can always revise that if we find we really want to log specifically to Sentry somewhere. In that case I'd also want to probe a bit into what that use case is really trying to do, and see if we can come up with an appropriate bit of API to describe that.

Reply:
(Very helpful to have this as a separate helper function, though! I'd waffled on whether to do that even when writing the current version, where the corresponding code is pretty simple.)

Reply:
That's fair. Done!